### PR TITLE
Fix vm panic under high loads

### PR DESCRIPTION
**Pull request by Liuhaai**
The multi threads panic under the high load
> thread '<unnamed>' panicked at 'object used with the wrong store', crates/wasmtime/src/store/data.rs:258:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
SIGABRT: abort
PC=0x7f698972bce1 m=7 sigcode=18446744073709551610

Other changes in the pr:
 - Upgrade wasmtime from v1 to v8 without changing interfaces
 - Limit one consumer to handle the event for each instance. The `maxMsgPerInstance` is limited to 5k
 - The result is responded once the event is inserted into the queue or timeout

### CODE REVIEW

File: pkg/modules/vm/wasmtime/instance.go [hunk 1]
```diff
@@ -18,14 +18,22 @@ import (
 	"github.com/machinefi/w3bstream/pkg/types/wasm"
 )
 
+const (
+	maxUint           = ^uint32(0)
+	maxInt            = int(maxUint >> 1)
+	maxMsgPerInstance = 5000
```
Comment (CoderZhi) [inline pkg/modules/vm/wasmtime/instance.go:24]: TODO (@Liuhaai): add into config

File: pkg/modules/vm/wasmtime/task.go [hunk 2]
```diff
@@ -39,23 +36,25 @@ func (t *Task) Arg() interface{} {
...
+	// case <-time.After(du):
+	// 	return &wasm.EventHandleResult{
+	// 		InstanceID: t.vm.ID(),
+	// 		Rsp:        nil,
+	// 		Code:       -1,
+	// 		ErrMsg:     "handle timeout",
+	// 	}
+	// case ret := <-t.Res:
+	// 	return ret
+	// }
```
Comment (CoderZhi) [inline pkg/modules/vm/wasmtime/task.go:50]: delete

File: pkg/modules/vm/wasmtime/task.go [hunk 3]
```diff
@@ -39,23 +36,25 @@ func (t *Task) Arg() interface{} {
...
 }
 
 func (t *Task) ID() string {
-	return fmt.Sprintf("%s::%s::%s", t.Subject(), t.vm.ID(), t.EventID)
+	panic("deprecated")
+	// return fmt.Sprintf("%s::%s::%s", t.Subject(), t.vm.ID(), t.EventID)
```
Comment (CoderZhi) [inline pkg/modules/vm/wasmtime/task.go:55]: delete

File: pkg/modules/vm/wasmtime/task.go [hunk 4]
```diff
@@ -39,23 +36,25 @@ func (t *Task) Arg() interface{} {
...
 }
 
 func (t *Task) Handle(ctx context.Context) {
-	t.Res <- t.vm.Handle(ctx, t)
+	// t.Res <- t.vm.Handle(ctx, t)
```
Comment (CoderZhi) [inline pkg/modules/vm/wasmtime/task.go:59]: delete

**Review by CoderZhi — APPROVED** [review]

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**